packages/k6-tests/common/uuid.js

```diff
@@ -0,0 +1,8 @@
+import { randomString } from 'https://jslib.k6.io/k6-utils/1.2.0/index.js';
+
+const random = len => randomString(len, '0123456789abcdef')
+export function uuid() {
```

Comment:
Is it an alternative to generate uuidv7? Can you add some comment here to explain why we are doing this?

Reply:
added comment to explain